Stripe/Project.swift

```diff
@@ -0,0 +1,181 @@
+import ProjectDescription
+
+let project = Project(
+    name: "Stripe",
+    options: .options(automaticSchemesOptions: .disabled),
+    packages: [
+        .remote(
+            url: "https://github.com/uber/ios-snapshot-test-case",
+            requirement: .upToNextMajor(from: "8.0.0")
+        ),
+        .remote(
+            url: "https://github.com/eurias-stripe/OHHTTPStubs",
+            requirement: .branch("master")
+        ),
+        .remote(url: "https://github.com/erikdoe/ocmock", requirement: .branch("master")),
+    ],
+    settings: .settings(
+        configurations: [
+            .debug(
+                name: "Debug",
+                xcconfig: "//BuildConfigurations/Project-Debug.xcconfig"
+            ),
+            .release(
+                name: "Release",
+                xcconfig: "//BuildConfigurations/Project-Release.xcconfig"
+            ),
+        ],
+        defaultSettings: .none
+    ),
+    targets: [
+        Target(
+            name: "StripeiOS",
+            platform: .iOS,
+            product: .framework,
+            productName: "Stripe",
+            bundleId: "com.stripe.stripe-ios",
+            infoPlist: "StripeiOS/Info.plist",
+            sources: "StripeiOS/Source/**/*.swift",
+            resources: "StripeiOS/Resources/**",
+            headers: .headers(
+                public: "StripeiOS/Stripe-umbrella.h"
+            ),
+            dependencies: [
+                .sdk(name: "Contacts", type: .framework),
```

Comment:
Are these necessary? I don't think this is a complete list of our dependencies, and everything is linked into the dyld_cache these days anyway.

Reply:
I was just copying what we had in the current project, we probably don't need them, I'll try removing them.